client/src/components/Kanban.js

```diff
@@ -1,201 +1,182 @@
 import { Settings } from "api"
+import Pie from "components/graphs/Pie"
 import LinkTo from "components/LinkTo"
+import { EngagementTrends } from "components/Trends"
 import moment from "moment"
 import PropTypes from "prop-types"
-import React from "react"
+import React, { useState } from "react"
 import { Button, Glyphicon, Panel } from "react-bootstrap"
-import Pie from "components/graphs/Pie"
-import { EngagementTrends } from "components/Trends"
 
-export default class Kanban extends React.Component {
-  static propTypes = {
-    tasks: PropTypes.array.isRequired,
-    columns: PropTypes.array.isRequired
-  }
+const Kanban = props => (
+  <div
+    style={{
+      display: "flex",
+      flexDirection: "row"
+    }}
+  >
+    {props.columns.map(column => {
+      return (
+        <Column
+          name={column.name}
+          taskUUIDs={column.tasks}
+          key={column.name}
+          tasks={props.tasks}
+        />
+      )
+    })}
+  </div>
+)
 
-  render() {
-    return (
-      <div
-        style={{
-          display: "flex",
-          flexDirection: "row"
-        }}
-      >
-        {this.props.columns.map(column => {
-          return (
-            <Column
-              name={column.name}
-              taskUUIDs={column.tasks}
-              key={column.name}
-              tasks={this.props.tasks}
-            />
-          )
-        })}
-      </div>
-    )
-  }
+Kanban.propTypes = {
+  tasks: PropTypes.array.isRequired,
+  columns: PropTypes.array.isRequired
 }
 
-class Column extends React.Component {
-  static propTypes = {
-    name: PropTypes.string.isRequired,
-    tasks: PropTypes.array.isRequired,
-    taskUUIDs: PropTypes.array.isRequired
-  }
+const Column = props => {
```

Comment:
Prefer `const Column = ({ name, tasks, taskUuids }) => {` instead of referring to `props.something` everywhere.